.circleci/config.yml

```diff
@@ -338,6 +338,10 @@ jobs:
           no-window: true
           restore-gradle-cache-prefix: v1a
           post-emulator-launch-assemble-command: 'echo "Emulator is ready"'
+      - run:
+          name: Disable gesture navigation
+          command: |
+            adb shell cmd overlay enable com.android.internal.systemui.navbar.threebutton
```

Comment:
Just wondering - shouldn't we test also gesture navigation scenarios?

Reply:
So the gesture navigation is just the way the user wants to have their device setup, and the tests fail because it tries to scroll from bottom, but ends up "closing" the app - video attached. So we use 3 button navigation instead. Using gestures, the app works fine, it's just for tests.